manage.py

```diff
@@ -50,6 +50,17 @@ def execute_sql_folder(path, processes):
             execute_sql_file(path)
 
 @manager.command
+def load_nicknames():
+    import pandas as pd
+    import sqlalchemy as sa
+    try:
+        table = sa.Table('ofec_nicknames', db.metadata, autoload_with=db.engine)
+        db.engine.execute(table.delete())
+    except sa.exc.NoSuchTableError:
+        pass
+    load_table(pd.read_csv('data/nicknames.csv'), 'ofec_nicknames')
+
+@manager.command
 def load_pacronyms():
```

Comment:
Thanks, I updated the patch and don't see this happening anymore. Can you verify that it's behaving as expected on your side as well?